main.go

```diff
@@ -196,6 +196,10 @@ func main() {
 		}
 	}
 
+	// Override default QPS (20) and Burst (30) to client configurations
+	hubCfg.QPS, hubCfg.Burst = tool.Options.ClientQPS, int(tool.Options.ClientBurst)
+	managedCfg.QPS, managedCfg.Burst = tool.Options.ClientQPS, int(tool.Options.ClientBurst)
```

Comment:
Using multiple assignments like this is unusual. I'd prefer each to have their own line.

I also wonder if it would be nice to be able to independently configure the QPS and Burst for the hub cluster client and the managed cluster client. I can imagine that a user would increase these settings in order to speed things up on the managed cluster, but would not want to open up the possibility of a massively increased number of requests to their hub cluster. Maybe another reviewer can provide input - we also don't want to have too many configuration options, because they could get hard to manage.